gmailservice/gmailservice.go

```diff
@@ -264,7 +268,7 @@ func ExtractHeader(gmail gmail.Message, field string) string {
 func GmailToMessage(gmail gmail.Message, inboxUrl string, downloaded time.Time) (store.Message, error) {
   // TODO: decode all of the fields, not just plain-text body
   date := time.Unix(gmail.InternalDate/1000, 0)
-  body := BodyText(gmail)
+  body := BodyText(gmail, "text/plain")
```

Comment:
We are making the assumption that anything without a text/plain representation does not need to be searched.  (From experience, these seem to be marketing emails).  Let me test this before merging, if this appears to be true, let's add a comment to this effect.

Reply:
@ultrasaurus How did testing go?